src/helpers/Url/Url.js

```diff
@@ -300,6 +300,7 @@ export function checkAndNormalizeUrl(url) {
         res.isValid = false;
       }
     }
+    if (res.url == undefined || res.url == null) res.isValid = false;
```

Comment:
```suggestion
    if (typeof res.url === "undefined" || res.url === null) res.isValid = false;
```

Reply:
Although I see that with modern JS it's possible to compare something to `undefined`.